SConstruct

```diff
@@ -1046,6 +1056,17 @@ if nan_works.strip() != "1":
         "either remove this option or add the '-fno-finite-math-only option'."
     )
 
+# Add conda library/include paths (if applicable)
+conda_prefix = os.environ.get("CONDA_PREFIX")
+if conda_prefix is not None:
+    if not selected_options & {"extra_inc_dirs", "extra_lib_dirs"}:
+        if os.name == "nt":
+            env.Append(CPPPATH=pjoin(conda_prefix, "Library", "include"),
+                    LIBPATH=pjoin(conda_prefix, "Library", env["libdirname"]))
+        else:
+            env.Append(CPPPATH=pjoin(conda_prefix, "include"),
+                    LIBPATH=pjoin(conda_prefix, env["libdirname"]))
+
```

Comment:
Love this change, so that we don't have to put `boost_inc_dir` into the settings anymore 🎉

Since it's just appending to the paths, I don't think it needs to be conditional on not having user values for those two variables. Also, we may want to log that this is happening.

Reply:
Yup. It appears to work on the GH Sundials runner also ...